Jenkinsfile.central

```diff
@@ -44,21 +44,26 @@ node {
                         credentialsId: 'SONATYPE_PASSWORD',
                         variable: 'SONATYPE_PASSWORD'],
                         [$class: 'StringBinding',
-                        credentialsId: 'PGP_HEX_KEY',
-                        variable: 'PGP_HEX_KEY'],
+                        credentialsId: 'GPG_KEY',
+                        variable: 'GPG_KEY'],
                         [$class: 'StringBinding',
-                        credentialsId: 'PGP_PASSPHRASE',
-                        variable: 'PGP_PASSPHRASE']]) {
+                        credentialsId: 'GPG_KEY_ID',
+                        variable: 'GPG_KEY_ID']]) {
         wrap([$class: 'AnsiColorBuildWrapper']) {
           sh """#!/bin/bash
             docker-compose \
-               -f "docker-compose.test.yml" \
-               run --rm --no-deps \
-               -e SONATYPE_USERNAME="${SONATYPE_USERNAME}" \
-               -e SONATYPE_PASSWORD="${SONATYPE_PASSWORD}" \
-               -e PGP_HEX_KEY="${PGP_HEX_KEY}" \
-               -e PGP_PASSPHRASE="${PGP_PASSPHRASE}" \
-               build "gitSnapshots ;release release-version ${params.RELEASE_TAG} with-defaults"
+                -f "${DIR}/../docker-compose.test.yml" \
+                run --rm --no-deps --entrypoint "bash -c" \
+                -e SONATYPE_USERNAME="${SONATYPE_USERNAME}" \
+                -e SONATYPE_PASSWORD="${SONATYPE_PASSWORD}" \
+                -e GPG_KEY_ID="${GPG_KEY_ID}" \
+                build "
+                    gpg --keyserver keyserver.ubuntu.com \
+                        --recv-keys ${GPG_KEY_ID} &&
+                    echo ${GPG_KEY} | base64 -d >signing_key.asc &&
+                    gpg --import signing_key.asc &&
+                    ./sbt ";sonatypeOpen ${BUILD_NUMBER};publish;sonatypeRelease"
```

Comment:
Instead of passing the release version here, we will determine use the tag returned by `git describe`.

See: https://github.com/geotrellis/geotrellis-server/blob/7510d1d102348d80773cedcd069527693e691870/scripts/cipublish#L21-L23

Reply:
Initially I was worried about relying on `git describe` for accurate version information in the context of this project, but perhaps it'll have all of the information it needs? Based on the issue template, it looks like we do `git flow finish` before invoking the Jenkins pipelines that publish to Maven Central, which makes me feel like it might be able to resolve the appropriate Git tag.

In the GeoTrellis Server reference you provided, the attempt to determine the release version occurs with the tagged commit checked out.

Reply:
We _should_ be able to rely on `git describe` since Jenkins checks out the tagged commit:

https://github.com/raster-foundry/raster-foundry/blob/74abb7d3142ef9e64a5b930f381ebc71c747a4a8/Jenkinsfile.central#L18-L26

Reply:
Gotcha. I missed that on the initial pass.